src/dragonball/Makefile

```diff
@@ -6,7 +6,7 @@ include ../../utils.mk
 
 PROJECT_DIRS := $(shell find . -name Cargo.toml -printf '%h\n' | sort -u)
 
-ifeq ($(ARCH), $(filter $(ARCH), s390x ppc64le))
+ifeq ($(ARCH), $(filter $(ARCH), s390x powerpc64le))
```

Comment:
I had missed that but it should be `powerpc64`.

Reply:
This change is just the consequence of the one in `utils.mk` and doesn't seem to have any impact. I guess it is ok then.